app/src/main/java/model/place/Category.kt

```diff
@@ -52,9 +53,15 @@ data class Category(
          */
         fun fromDocument(document: DocumentSnapshot): Category? {
             val id = document.id.toInt()
-            val en = document["en"] as? String ?: ""
-            val fr = document["fr"] as? String ?: ""
-            return Category(id, en, fr)
+            val en = document["en"] as? String
+            val fr = document["fr"] as? String
+
+            return if (en != null && fr != null) {
+                Category(id, en, fr)
+            } else {
+                Timber.tag("LoadCategories").e(Exception("Error parsing Category with id $id. en: $en, fr: $fr"))
```

Comment:
Optional, but another way of logging, which I did for parsing, is to create an inner function:

```
fun logIncomplete(message: String): Category? {
    Timber.tag("LoadCategories").e(Exception(message))
    return null
}
```

So you can do something like 

```
val en = document["en"] as? String ?: return logIncomplete("Null en")
```